project.clj

```diff
@@ -8,14 +10,15 @@
 
   :min-lein-version "2.7.1"
 
-  :parent-project {:coords [puppetlabs/clj-parent "5.2.9"]
+  :parent-project {:coords [puppetlabs/clj-parent "7.0.1"]
                    :inherit [:managed-dependencies]}
 
   :dependencies [[puppetlabs/pcp-common "1.3.5" :exclusions [org.tukaani/xz]]
-
-                 [stylefruits/gniazdo nil :exclusions [org.eclipse.jetty.websocket/websocket-client]]
                  ;; We only care about org.eclipse.jetty.websocket/websocket-client
-                 [puppetlabs/trapperkeeper-webserver-jetty9]
+                 [com.puppetlabs/trapperkeeper-webserver-jetty10 "1.0.2"]
+                 [org.eclipse.jetty.websocket/websocket-jetty-client ~jetty-version]
+                 [org.eclipse.jetty.websocket/websocket-jetty-api ~jetty-version]
+                 [hato "0.9.0"]
```

Comment:
Should this be in the dev-dependencies?  Is it a hard requirement for the client?

Reply:
Is it even used? I think it can be removed.

Reply:
Removed